docs/reference/glossary.md

```diff
@@ -442,244 +378,240 @@ A SpatialOS project includes (but isn't limited to):
-it can update [properties](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#property), send and handle [commands](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#command) and trigger [events](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#event).
+Client-workers are typically tasked with visualizing what’s happening in the [SpatialOS world](#spatialos-world) and handling player input.
 
-See also the [Workers in the GDK]({{urlRoot}}/reference/concepts/worker) documentation.
+### Server-worker
 
-> Related:
+<span style="display: block;margin-top: -20px;font-size: 10pt;"><i>Also known as “managed worker”, “server-side worker”.</i></span>
+
+A server-worker is a [worker instance](#worker) whose lifecycle is managed by SpatialOS. When running a [deployment](#deploying), the SpatialOS Runtime starts and stops server-workers based on your chosen [load balancing](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#load-balancing) configuration.
+
+Server-workers are typically tasked with running game logic and physics simulation.
+
+> **Related:**
 >
-> * [Concepts: Workers and load balancing](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/concepts/workers-load-balancing)
+> * [Managed worker (server-worker) launch configuration](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/worker-configuration/launch-configuration#managed-worker-launch-configuration)
 
-### Worker attribute
+## Worker attribute
 
-Worker attributes are used to denote a [worker’s](#worker) capabilities. The [SpatialOS runtime](#spatialos-runtime) uses these attributes to delegate [authority](#authority) over [SpatialOS components](#spatialos-component) in combination with the defined [ACL](#access-control-list-acl). A worker’s attributes are defined in its [worker configuration JSON](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/worker-configuration/bridge-config#worker-attribute-sets).
+Worker attributes are used to denote a [worker’s](#worker) capabilities. 
 
-> Related:
+The [SpatialOS Runtime](#spatialos-runtime) uses these attributes to delegate [authority](#authority) over [SpatialOS components](#spatialos-component) in combination with the defined [ACL](#access-control-list-acl). 
+
+A worker’s attributes are defined in its [worker configuration JSON](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/project-layout/bridge-config).
+
+> **Related:**
 >
 > * [Worker attributes](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/glossary#worker-attribute)
 > * [Bridge configuration](https://docs.improbable.io/reference/<%(Var key="worker_sdk_version")%>/shared/worker-configuration/bridge-config#worker-attribute-sets)
```

Comment:
link is broken

Reply:
https://docs.improbable.io/reference/13.8/shared/project-layout/launch-configuration#managed-worker-launch-configuration